galasa-managers-parent/galasa-managers-logging-parent/dev.galasa.elasticlog.manager/src/main/java/dev/galasa/elasticlog/internal/package-info.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * ElasticLog Manager
+ * 
+ * @galasa.manager ElasticLog
+ * 
+ * @galasa.release.state ALPHA - This Manager has been written to provide a way of exporting test data to ElasticSearch.  It has the bare 
+ * minimum code necessary to do that.  It has not been extensively tested.  The TPI of the LoggingManager interface is subject to change. 
+ * It can be used within tests.
+ * 
+ * @galasa.description
+ * 
+ * This Manager provides an interface to allow test data to be exported to an ElasticSearch endpoint. This allows for test data to be 
+ * visualised and logged. 
+ * <br><br>
+ * As an absolute minimum, the CPS properties <code>elasticlog.endpoint.address</code> and <code>elasticlog.endpoint.index</code> 
+ * must be provided with <code>elasticlog.local.run.log</code> set to true to active the manager locally.
```

Comment:
change <code>elasticlog.local.run.log</code> set to true to active the manager locally.
to
By default, this manager will only log automated tests, to enable logging from locally run tests, <code>elasticlog.local.run.log</code> must be set to true.